Xamarin.Essentials/Launcher/Launcher.ios.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Threading.Tasks;
+using Foundation;
+using UIKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Launcher
+    {
+        static Task<bool> PlatformCanOpenAsync(Uri uri)
+        {
+            return Task.FromResult(UIApplication.SharedApplication.CanOpenUrl(new NSUrl(uri.ToString())));
+        }
+
+        static async Task PlatformOpenAsync(Uri uri)
+        {
+            await UIApplication.SharedApplication.OpenUrlAsync(new NSUrl(uri.ToString()), new UIApplicationOpenUrlOptions());
```

Comment:
Should we make the string to NSUrl and extension method somewhere?

Reply:
Probably, I guess many other people use uri.ToString(). Having conversion operations via extension methods for every platform is advisable. I would suggest putting this in a new pr though.